libraries/eosiolib/contracts/eosio/privileged.hpp

```diff
@@ -185,6 +212,217 @@ namespace eosio {
     */
    void get_blockchain_parameters(eosio::blockchain_parameters& params);
 
+   /**
+    *  Helper structure for host functions set_parameters_packed and get_parameters_packed
+    *  @ingroup privileged
+    */
+   struct params_object{
+      std::string packed;
+
+      params_object() = default;
+
+      template<typename T>
+      params_object(T v){
+         (*this)(v);
+      }
+
+      params_object(const std::string& params) : packed(params) {}
+
+      bool operator == (const params_object& pp) const{
+         return packed == pp.packed;
+      }
+
+      params_object& operator () (unsigned_int val){
```

Comment:
What is the motivation to use the 'call operator' instead of the << operator?  It is a bit odd.

Reply:
I removed this code, as the get_parameters_packed and set_parameters packed host functions were implemented in a different way.